docs/user/cli-installation.rst

```diff
@@ -112,9 +112,9 @@ it installed, you can run
 .. code-block:: console
 
   conda create -y -n renku python=3.6
-  $(conda env list | grep renku | awk '{print $2}')/bin/pip install -e git+https://github.com/SwissDataScienceCenter/renku-python.git#egg=renku
+  $(conda env list | grep 'renku ' | awk '{print $2}')/bin/pip install -e git+https://github.com/SwissDataScienceCenter/renku-python.git#egg=renku
```

Comment:
We should do `grep '^renku\s'` so it also covers the case of an env being name e.g. `my-renku` as well as `renku-env`